.eslintrc

```diff
@@ -9,7 +9,8 @@ env:
 
 rules:
   no-console: 0  # This project purposely uses console.
-
+  space-before-function-paren: ["error","never"]
+  padded-blocks: ["error", "never"]
```

Comment:
I'm not sure why you think this fixes the style. This change actually disables rules.